src/data_sets.h

```diff
@@ -3756,6 +3756,8 @@ typedef struct
     double baseline[3];
     double max_baseline_error;
     double reset_baseline_error;
+    /** velocity constraint in compassing mode {var before fix, var after fix} (m^2/s^2) **/
```

Comment:
This should probably be added at the end, so it doesn't break compatibility with different versions of the SDK/firmware.

Reply:
moved it